### DISCUSSION TITLE
AMT or vPro or what?

### DISCUSSION THREAD

**Discussion by simpsonetti** [Ylianst/MeshCentral#5639] [Q&A]
Hi there, 
some kind question: we have a dell contract and get all our devices from them. i didn't notice that my testlaptop doesn't have vPro. But what do i configure in dell bios ? Is this a small Version of vPro, AMT or what? 
Here in meshcentral it says:

Intel® AMT | Activated ACM, v15.0.45, Invalid Credentials

Hmmm.... 
Some hints?

**Comment by Ganeshr93** [comment]
On the AMT device, try running meshcmd amtinfo or sudo ./meshcmd amtinfo and see the output. It should spit out more details about the device.
